assets/src/components/map.tsx

```diff
@@ -478,17 +476,36 @@ export const FollowerStatusClasses = (
 
 export const MapFollowingPrimaryVehicles = (props: Props) => {
   const state = useInteractiveFollowerState(),
-    { shouldFollow, setShouldFollow } = state
+    { shouldFollow } = state
 
   const positions: LatLng[] = props.vehicles.map(vehicleToLeafletLatLng)
 
-  useEffect(
-    () => setShouldFollow(true),
-    [props.followerResetKey, setShouldFollow]
+  return (
+    <Map {...props} stateClasses={FollowerStatusClasses(shouldFollow)}>
```

Comment:
comment(non-blocking): I just noticed that if `props` specifies `stateClasses` we're discarding those here. This isn't an issue _now_ but I wanted to make others aware of this logic (since I originally wrote it and I didn't mean to miss that).